### PR TITLE
ED-1529 export opportunity view

### PR DESCRIPTION
**Pull request by sdonk**
https://uktrade.atlassian.net/browse/ED-1529

Confirmation page preview:

screen shot 2017-07-11 at 09 57 10

### CODE REVIEW

File: exportopportunity/views.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    success_template = 'export-opportunity-success.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        if not settings.FEATURE_EXPORT_OPPORTUNITY_LEAD_GENERATION_ENABLED:
+            raise Http404()
+        super().dispatch(request, *args, **kwargs)
+
+    def form_valid(self, form):
+        response = api_client.exportopportunity.create_opportunity(
+                form_data=form.cleaned_data
```
Comment (richtier) [inline exportopportunity/views.py]: funky indentation

File: makefile [hunk 2]
```diff
@@ -55,6 +55,10 @@ DOCKER_SET_DEBUG_ENV_VARS := \
 	export DIRECTORY_UI_SUPPLIER_FEATURE_MORE_INDUSTRIES_BUTTON_ENABLED=true; \
 	export DIRECTORY_UI_SUPPLIER_FEATURE_SPORTS_INFRASTRUCTURE_ENABLED=true; \
 	export DIRECTORY_UI_SUPPLIER_FEATURE_COMPANY_SEARCH_VIEW_ENABLED=true
+	export DIRECTORY_UI_SUPPLIER_FEATURE_COMPANY_SEARCH_VIEW_ENABLED=true; \
+	export DIRECTORY_UI_SUPPLIER_COVERALLS_REPO_TOKEN=$$COVERALLS_REPO_TOKEN; \
```
Comment (richtier) [inline makefile]: is this wanted?

File: exportopportunity/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        'bid_timing': '2017-09-09',
+        'full_name': 'Testo Useri',
+        'email_address': 'test@foo.com',
+        'company_name': 'Acme'
+    }
+
+    response = client.post(reverse('export-opportunity'), data=data)
+
+    assert response.status_code == 200
+    assert response.template_name == [expected_template_name]
```
Comment (sdonk) [inline exportopportunity/tests/test_views.py]: I should have probably mocked the client library and assert on the mock

File: makefile [hunk 4]
```diff
@@ -55,6 +55,10 @@ DOCKER_SET_DEBUG_ENV_VARS := \
...
+	export DIRECTORY_UI_SUPPLIER_FEATURE_INDUSTRIES_TRANSLATIONS_ENABLED=true; \
```
Comment (richtier) [inline makefile]: we no longer want this, and you can remove it form settings.py too.

File: makefile [hunk 5]
```diff
@@ -110,6 +114,9 @@ DEBUG_SET_ENV_VARS := \
 	export FEATURE_SPORTS_INFRASTRUCTURE_ENABLED=true; \
 	export FEATURE_COMPANY_SEARCH_VIEW_ENABLED=true; \
 	export DISABLED_LANGUAGES_INDUSTRIES_PAGE=zh-hans,fr,pt-br
+	export FEATURE_INDUSTRIES_TRANSLATIONS_ENABLED=true; \
+	export FEATURE_INDUSTRIES_TRANSLATIONS_ENABLED=false; \
```
Comment (richtier) [inline makefile]: duplicate. we don't want it anyway - can remove it.

File: makefile (hunk 2 above)
Comment (sdonk) [inline makefile]: I wasn't sure, it was in your branch. I'll remove everything

**Pull request merged**